haskell/sbp.cabal

```diff
@@ -1,5 +1,5 @@
 name:                  sbp
-version:               2.2.3
+version:               2.2.3-dirty
```

Comment:
@mfine Does it mean that the process is, first, commit my changes in .yaml and python tests if any, second, generate other code and finally make one more commit with generated files?
I usually make change, generate code and commit everything at once.